api/src/client/dynamics/license-status/formatRegulatedBusinessDynamicsApplications.test.ts

```diff
@@ -20,17 +19,4 @@ describe("formatRegulatedBusinessDynamicsApplications", () => {
       },
     });
   });
-
-  it("throws DUPLICATE_LICENSE_TYPE_ERROR error when there is a duplicate professionNameAndLicenseType", () => {
-    const checklistResult1 = generateLicenseStatusChecklistResult({
-      professionNameAndLicenseType: "duplicate profession name and license type",
-    });
-    const checklistResult2 = generateLicenseStatusChecklistResult({
-      professionNameAndLicenseType: "duplicate profession name and license type",
-    });
-
-    expect(() => formatRegulatedBusinessDynamicsApplications([checklistResult1, checklistResult2])).toThrow(
-      DUPLICATE_LICENSE_TYPE_ERROR
-    );
-  });
```

Comment:
[sand] - I wonder if it's worth expecting the inverse now as a happy path case to communicate to future devs that there can be duplicates.

Reply:
the duplicates get over written, i added a test. thanks for the feedback